sample_data/iowa_liquor_sales_dataprep.ipynb

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
What is this notebook for? Do we need to keep it?

Reply:
It prepares the demo dataset, it makes sense to include it.